README.md

```diff
@@ -4,4 +4,11 @@ This repository is for showing how Git and GitHub work
 
 ## Purpose
 
-Sample website with plenty of files for demos
\ No newline at end of file
+Sample website with plenty of files for demos
+
+Updated the readme file since it was already there.
+## More content
+
+Sample website with plenty of files for demos
```

Comment:
```suggestion

Updated the readme file since it was already there.
## More content

Sample website with plenty of files for demos

This is my suggestion

```